conf.d/main

```diff
@@ -9,7 +9,12 @@ apt install --assume-yes wireguard
 # webserver related
 mkdir -p /var/www/wireguard/logs
 chown www-data:www-data /var/www/wireguard/logs
-lighty-disable-mod ssl
+
+service lighttpd restart
```

Comment:
This line should be required. I'm guessing that you added it for debugging/testing and just forgot to remove it?

Reply:
Did you mean to say these lines should not be required? I can replace these lines with the one you suggested to test lighttpd capabilities during conf.d build.